docs/manual/keyspace-notifications.md

```diff
@@ -82,7 +82,7 @@ will be delivered regardless of the rest of the string.
 For instance to enable just Key-space events for lists, the configuration
 parameter must be set to `Kl`, and so forth.
 
-The string `KEA` can be used to enable every possible event.
+The string `KEAm` can be used to enable every possible event.
```

Comment:
now that we have a notification for new keys (https://github.com/redis/redis/pull/10512), this will get outdated again.
how about keeping the `KEA` and instead change the text to so that it doesn't say "every possible event"?